src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/ImportOrchestratorJobTests.cs

```diff
@@ -52,12 +52,30 @@ public async Task GivenAnOrchestratorJob_WhenResumeFromFailure_ThenJobShouldBeCo
             await VerifyCommonOrchestratorJobAsync(105, 6, 10);
         }
 
+        [Fact]
+        public async Task GivenAnOrchestratorJob_WhenAllResumeFromFailure_ThenJobShouldBeCompleted()
+        {
+            await VerifyCommonOrchestratorJobAsync(105, 6, 105);
+        }
+
         [Fact]
         public async Task GivenAnOrchestratorJob_WhenResumeFromFailureSomeJobStillRunning_ThenJobShouldBeCompleted()
         {
             await VerifyCommonOrchestratorJobAsync(105, 6, 10, 5);
         }
 
+        [Fact]
+        public async Task GivenAnOrchestratorJob_SetSomeJobCancelled()
```

Comment:
>GivenAnOrchestratorJob_SetSomeJobCancelled [](http://example.com/codeflow?start=26&length=42)

The pattern is Given_When_then

Reply:
Ok, I will check and modify all function name.